### PR TITLE
fix #1614 scroll issue for test set selector

### LINKED ISSUE #1614: Suite selector gets overlayed and scrolls incorrectly
With a long list of tests, the suite selector gets drawn over and then will scroll improperly.

image

### PR DESCRIPTION
**Pull request by coinzdude**
fix to address display issue for test selector in TestDrawer with longer lists of tests. Move selector into subheader.

### CODE REVIEW

File: packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Tests/TestsDrawer.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import EditorToolbar from '../../components/Drawer/EditorToolbar'
 import RenamableListItem from '../../components/Drawer/RenamableListItem'
 import TestNewDialog from './TestNewDialog'
 import { StateShape } from '@seleniumhq/side-api'
+import { Stack } from '@mui/material'
```
Comment (toddtarsi) [inline packages/selenium-ide/src/browser/windows/ProjectEditor/tabs/Tests/TestsDrawer.tsx]: @coinzdude - Can you instead import this as:

```
import Stack from '@mui/material/Stack'
```

It's a tree shaky thing. Performance in the app is terrible but I still like to pretend I care about an extra 250 kb every now and again.

**Review by toddtarsi — COMMENTED** [review]
@coinzdude - Nice work, PR looks good. Amazing that one little component would resolve the issue. Would you do the following remaining steps:

1. Tweak that import to use a specific component
2. Increment the selenium-ide version [here](https://github.com/SeleniumHQ/selenium-ide/blob/trunk/packages/selenium-ide/package.json#L3)
3. Merge this to trunk and it will automatically build a new version of the IDE. 

Thanks for using the tool, finding the bugs, and fixing them yourself. Glad to have ya back

**Comment by toddtarsi** [comment]
@coinzdude - Hey, I just encountered that same bug locally. Stack actually makes perfect sense. That select element is like totally transparent. Wild!

**Comment by toddtarsi** [comment]
Hm. It seems like it scrolls kinda funny. Does stack resolve that too?
Screenshot 2023-04-20 at 8 49 18 PM

**Comment by coinzdude** [comment]
> Hm. It seems like it scrolls kinda funny. Does stack resolve that too? 

This seems to be CSS based and defined at some level by the Drawer control. I wasn't able to find how scrollbar is controlled here.

I wanted it to stick too, but at least it wasn't floating in the middle or over other text.  Open to ideas on it!

**Comment by coinzdude** [comment]
Just committed tweaks, let me know if this looks good

**Comment by toddtarsi** [comment]
Perfect, okay I'll merge it now. The CI is my bad, I'll fix that soon.

**Pull request merged**